src/main/java/eu/dissco/orchestration/backend/service/KafkaPublisherService.java

```diff
@@ -0,0 +1,53 @@
+package eu.dissco.orchestration.backend.service;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import eu.dissco.orchestration.backend.domain.CreateUpdateDeleteEvent;
+import java.time.Instant;
+import java.util.UUID;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class KafkaPublisherService {
+
+  private final KafkaTemplate<String, String> kafkaTemplate;
+  private final ObjectMapper mapper;
+
+  public void publishCreateEvent(String id, JsonNode object, String subjectType)
+      throws JsonProcessingException {
+    var event = new CreateUpdateDeleteEvent(
+        UUID.randomUUID(),
+        "create",
+        "orchestration-backend",
+        id,
+        subjectType,
+        Instant.now(),
+        object,
+        null,
+        "Object is newly created");
+    log.info("Publishing new create message to queue: {}", event);
+    kafkaTemplate.send("createUpdateDeleteTopic", mapper.writeValueAsString(event));
+  }
+
+  public void publishUpdateEvent(String id, JsonNode newRecord, JsonNode jsonPatch,
+      String subjectType) throws JsonProcessingException {
+    var event = new CreateUpdateDeleteEvent(
+        UUID.randomUUID(),
+        "update",
+        "orchestration-backend",
+        id,
+        subjectType,
+        Instant.now(),
+        newRecord,
+        jsonPatch,
+        "Object has been updated");
+    log.info("Publishing new update message to queue: {}", event);
+    kafkaTemplate.send("createUpdateDeleteTopic", mapper.writeValueAsString(event));
+  }
```

Comment:
Why not send delete Event?

Reply:
Yes good question was think this myself yesterday. No service sends delete's yet. Think we should start but need to figure out what they should look like. Is it just, Delete with no body and no jsonpatch. Or do we put in the last version... need to think on it

Reply:
I will add a new jira ticket for this as it requires some thought work and some implementation work. 
https://naturalis.atlassian.net/browse/DD-570